### ISSUE TITLE
ML Project 

### ISSUE THREAD

**Issue by NiladriXX** [NiladriXX/QTECHSQUBITS#2]
# House Price Prediction

import numpy as np
import pandas as pd
from sklearn.linear_model import LinearRegression

# Load the data
data = pd.read_csv("housing.csv")

# Split the data into features and target
features = data.drop("SalePrice", axis=1)
target = data["SalePrice"]

# Create the linear regression model
model = LinearRegression()

# Train the model
model.fit(features, target)

# Predict the house prices
predictions = model.predict(features)

# Evaluate the model
print(model.score(features, target))
